Automattic/wp-calypso
# Cancelation survey: ask import specific questions

#### Changes proposed in this Pull Request

* Updates the "remove purchase" survey to have an additional question about the import quality when the associated site has done an import

#### Testing instructions

* Apply and follow the instructions on D33115-code.
* Complete an import and make a purchase (the order should not matter).
* Visit `/me/purchases` and select your purchase.
* Select "Cancel Subscription and Refund".
* Select "Cancel Subscription".
* Verify that in addition to the 2 default questions, a third question about importing appears.
![image](https://user-images.githubusercontent.com/363749/65989090-a2b71400-e44e-11e9-93b7-3b4b272d8d21.png)
* Answer the question, and verify that you are able to continue. Verify that your answer is recorded (see D33404-code).
